parts/linux/cloud-init/artifacts/bind-mount.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+# Bind mount kubelet to ephemeral storage on startup, as necessary.
+#
+# This fixes an issue with kubelet's ability to detect allocatable
+# capacity for Node ephemeral-storage. On Azure, ephemeral-storage
+# should correspond to the temp disk if a VM has one. This script makes
+# that true by bind mounting the temp disk to /var/lib/kubelet, so
+# kubelet thinks it's located on the temp disk (/dev/sdb). This results
+# in correct calculation of ephemeral-storage capacity.
+
+{{if eq GetKubeletDiskType "Temporary"}}
+MOUNT_POINT="/mnt/aks"
```

Comment:
when do we create this mount point?

Reply:
discussed offline, but L23 and /mnt will already exist and be on the temporary disk.